platforms/gke/base/core/deploy.sh

```diff
@@ -17,19 +17,20 @@ set -o errexit
 
 start_timestamp=$(date +%s)
 
-terraservices=("networking" "container_cluster" "container_node_pool" "gke_enterprise/fleet_membership" "workloads/kueue")
 # Disable gke_enterprise/servicemesh due to b/376312292
-#terraservices=("networking" "container_cluster" "container_node_pool" "gke_enterprise/fleet_membership" "gke_enterprise/servicemesh" "workloads/kueue")
+declare -a terraservices=${TERRASERVICES:-("networking" "container_cluster" "container_node_pool" "gke_enterprise/fleet_membership" "workloads/kueue")}
```

Comment:
Can we rename `TERRASERVICES` to something more specific about the core platform?

Reply:
Yes, we can rename it to `CORE_TERRASERVICES`